requests-and-responses/requests-and-responses.md

```diff
@@ -7,7 +7,7 @@
 - [Command Line](/command-line/command-line-interface.md)
 
 ### Motivation
-As a web developer, your code will run on and interact with the internet! Understanding how the internet actually works will help you know what your code is doing and how your code interacts with other code on the internet.
+As a web developer, your code will run on and interact with the internet! Understanding how the internet actually works will help you know what your code is doing and how your code interacts with other code on the internet. 
```

Comment:
adding a space?  not quite enough to get gssoc points, I'm afraid.

Reply:
It wasn't intentional. just a key was pressed. sorry.

Reply:
That’s fine. I highly recommend looking at the git diff and adding files one at a time, so you know exactly what you are committing. It may feel slow, but it’s a lot faster than making mistakes and having to undo them.

Reply:
Also, on a developer team, people don’t like to see this... it’s ‘noise’, and gets in the way of the real changes they’re trying to look at.

Reply:
okay. I will take care of this next time.

Reply:
Can you take care of it this time?